distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc

```diff
@@ -78,85 +62,127 @@ By default, the response body will include the XML representation of the Metacar
-Content-Type:
+Creates a generic resource metacard with the provided XML as content for the `metadata` XML field
+in the metacard.
 
+.Request with Missing XML Prologue
+The following is an example request with a missing XML prologue sent to the Catalog ReST endpoint.
 
-------WebKitFormBoundary7MA4YWxkTrZu0gW--
+.Missing XML Tag Example
 ----
+<metacard xmlns="urn:catalog:metacard" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:smil="http://www.w3.org/2001/SMIL20/" xmlns:smillang="http://www.w3.org/2001/SMIL20/Language" gml:id="3a59483ba44e403a9f0044580343007e">
+  <type>ddf.metacard</type>
+  <string name="title">
+    <value>Test REST Metacard</value>
+  </string>
+  <string name="description">
+    <value>Vestibulum quis ipsum sit amet metus imperdiet vehicula. Nulla scelerisque cursus mi.</value>
+  </string>
+</metacard>
+----
+
+Metacard is created successfully
+
+.Request with Non-XML Data
+The following is an example request with non-XML data sent to the Catalog ReST endpoint.
 
-===== Metacard Transforms with the REST CRUD Endpoint
+.Non-XML data Example
+----
+title: Non-XML title
+id: abc123
+----
 
-The `read` operation can be used to retrieve metadata in different formats.
+Metacard will be created and the content will stored in the `metadata` field.
 
-. Install the appropriate feature for the desired transformer. If desired transformer is already installed such as those that come out of the box (`xml,html,etc`), then skip this step.
-. Make a read request to the REST URL specifying the catalog id.
-. Add a transform query parameter to the end of the URL specifying the shortname of the transformer to be used (e.g., `transform=kml`).
+.Request with Invalid Transform
+Testing valid data with an invalid `transform=invalid` appended to the POST URL: `{public_url}/services/catalog?transform=blah`
 
-.Example Metacard Transform
-[source,http]
+.Valid data with an invalid `?transform=invalid`
 ----
-${public_url}/services/catalog/<metacardId>?transform=<TRANSFORMER_ID>
+<?xml version="1.0" encoding="UTF-8"?>
+<metacard xmlns="urn:catalog:metacard" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:smil="http://www.w3.org/2001/SMIL20/" xmlns:smillang="http://www.w3.org/2001/SMIL20/Language" gml:id="3a59483ba44e403a9f0044580343007e">
+  <type>ddf.metacard</type>
+  <string name="title">
+    <value>Test REST Metacard</value>
+  </string>
+  <string name="description">
+    <value>Vestibulum quis ipsum sit amet metus imperdiet vehicula. Nulla scelerisque cursus mi.</value>
+  </string>
+</metacard>
 ----
 
-[TIP]
-====
-Transforms also work on read operations for metacards in federated sources.
-\http://<DISTRIBUTION_HOST>:<DISTRIBUTION_PORT>/services/catalog/sources/<sourceId>/<metacardId>?transform=<TRANSFORMER_ID>
-====
+A HTTP 400 is returned and the following response body is returned. The specific error is logged
+in the error log.
```

Comment:
pretty nit-picky, but I'd like to have the line breaks on the sentences where possible.